plugins/osdn/common.go

```diff
@@ -29,6 +29,7 @@ type OvsController struct {
 	Registry        *Registry
 	localIP         string
 	localSubnet     *api.Subnet
+	clusterNet      *net.IPNet
```

Comment:
It feels a bit strange to me to use a field to pass this argument into the validation methods.  Is StartMaster always guaranteed to be called before the validation methods?